samples/error.json

```diff
@@ -1,10 +1,17 @@
 {
-  "stages": [
-    {
-      "name": "org.osbuild.error"
+  "sources": {
+    "org.osbuild.files": {
+      "urls": {}
+    }
+  },
+  "pipeline": {
+    "stages": [
+      {
+        "name": "org.osbuild.error"
+      }
+    ],
+    "assembler": {
+      "name": "org.osbuild.noop"
```

Comment:
Is this just a result of the converter script? Because you could just omit "sources" and it would have the same effect. But nothing crucial.

Reply:
Yeah, exactly.